### PR TITLE
Leaderboard - API Implementation

### PR DESCRIPTION
**Pull request by ReshyResh**
# New features: 
  - Function to make POST request to game API with user input.
  - Function to populate the list fetching data from the API.

# Live demo
### You can view the website by clicking [here](https://reshyresh.github.io/Leaderboard-API-Js/)

### CODE REVIEW

File: src/sendData.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default (name, score) => {
```
Comment (rloterh) [inline src/sendData.js:1]: - [ ] The `promise` feature is not well implemented you _sending_ and _receiving_ methods. Kindly ensure to use `async` and `await` with the fetch method to send and receive data, so as to handle the data efficiently

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Robert, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Sending and retrieving data is working as expected
- Nice UI design
- No linter errors


## Required Changes ♻️
- [ ] _Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/sendData.js (hunk 1 above)
Comment (rloterh) [reply src/sendData.js]: This issue is not properly resolved

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Paka and Yannick-Noel, 

You did great on this project, and for making the changes requested in the previous review!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚥 🚦 

## Good Points 👍  
- No linter errors

## Required Changes ♻️
- [ ] _Kindly check the comments above, made under the reviews of the previous code reviewer(s) and stating **not properly resolved**. Kindly take your time to correctly review those issues outlined by the previous code reviewers, and please check to see that the changes are working as intended._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

This is an additional review as a result of the second opinion request.

The review was valid, however, the CR wasn't informative enough, I'll make sure to pass a note to reduce these cases :)

But it looks like you solved the issue, please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rloterh** [comment]
**Hello Robert!**

I've just realized I missed one very important point in my second review. I missed your changes in my cloned version.

I'm sorry I couldn't point that out to you during my review. Kindly note, this was not on purpose 😊 

Happy coding! 🍷 💻 

Cheers!

**Pull request merged**